spec/exporters/formatters/utaac_decision_indexable_formatter_spec.rb

```diff
@@ -29,7 +28,6 @@
   include_context "schema with humanized_facet_value available"
 
   it_behaves_like "a specialist document indexable formatter"
-  it_behaves_like "a tribunal decision indexable formatter"
```

Comment:
Same comment about this shared behaviour - it provided coverage of some functionality that's now slightly different, but not entirely gone - could we add a new set of shared examples and share it with `EmploymentTribunalDecisionIndexableFormatter` ?